Sources/docc-gpt/DoccGPTRunner.swift

```diff
@@ -21,8 +21,8 @@ struct DoccGPTRunner {
 
   // MARK: Private
 
-  /// The base URL for the OpenAI API.
-  private let baseURL = URL(string: "https://api.openai.com/v1/completions")!
+  /// The URL for the OpenAI API.
+  private let apiURL = URL(string: "https://api.openai.com/v1/completions")!
```

Comment:
This feels like a perfect use case for [`URL(staticString:)`](https://www.swiftbysundell.com/articles/constructing-urls-in-swift/#static-urls).

Reply:
It definitely is. Since there is only a single URL in the entire codebase, the force unwrap didn't bother me though lol 😄